### DISCUSSION TITLE
I've encountered a problem running a python file and the command 'python.execInTerminal-icon' is not found

### DISCUSSION THREAD

**Discussion by AvoFA** [coder/code-server#6226] [Q&A]
I've encountered a problem launching a Python file which gives the command 'python.execInTerminal-icon' not found. I've tried to rollback the Python extension and reinstall it but the result is the same. I also get the error when I press a keyboard shortcut to open the file but I can't get anything going by pressing a mouse button or opening the terminal. The problem came up while I was working, spontaneously.

**Comment by code-asher** [comment]
There might be some clues in the browser console logs and the logs in `~/.local/share/code-server/logs`.  If you post them I will take a look.

Usually this means the extension failed to activate and there should be an error log about that.
